src/root_files/_common/404.html

```diff
@@ -11,18 +11,17 @@
     }
 
     var langs = {
-        // TODO: uncomment if we enable de again
-        // 'de': 'de|at|li',
+        // TODO: uncomment if we enable id, ko and pt languages
+        // 'id': 'id',
+        // 'ko': 'kr',
+        // 'pt': 'br|mz|ao|pt|gw|pg|cv|st',
```

Comment:
`id`, `ko` and `pt` languages are not supported on the platform despite translation is available. This was causing the page get translated into these languages if the URL contained them while the EN lang was still shown as currently selected in language settings.